public/content/lessons/2018/fourier-transforms/index.mdx

```diff
@@ -6,4 +6,343 @@ video: spUNpyF58BY
+
+<Figure image="centered.png" width="100%" />
+
+The motive for using purely positive sine waves as our starting examples in this post is that negative values are a little weirder to think about as you wind up the graph. But the core logic stays the same in either case.
+
+At this stage, we can call this plot tracking the center of mass of our wound up graph the "Almost Fourier transform" of the original signal. It allows for us to pick out the frequency of the signal by seeing where the spike is. Next, suppose we take a different signal, like one with a lower frequency of 2 beats per second. We could do the same thing and find a peak at 2 cycles per second, as shown here:
+
+<Figure
+  image="2ps.png" width="100%"
+/>
+
+The most interesting aspect of this machine is that it will allow us to decompose any arbitrary signals into it's constituent waveforms. To show this, let's combine the 2 and 3 frequency waves and do the same operation: tracking the center of mass coordinate and look for peaks in the winding frequency plot. This is shown below:
+
+<Figure
+  image="full_machine.png"
+  video="full_machine.mp4"
+  width="100%"
+/>
+
+Here, we see 2 different frequencies at both 2 and 3 beats per second, which is exactly the same as the two "Almost Fourier transformed" plots added together. Simply put, the sum of the two "Almost Fourier transformed" signals is the same as the "Almost Fourier transform" of the two summed together, as shown here:
+
+<Figure
+  image="almost.png" width="100%"
+/>
+
+The way a mathematician might phrase this is that our "Almost Fourier transform" operation is *linear*. The curious among you may want to pause for a moment to reflect on why this is true. It ultimately stems from the fact that the x-coordinate of the sum of two vectors is the sum of their x-coordinates. This, by the way, is why we had it track the x-coordinate of the center of mass of the graph, rather than tracking its distance from the origin. For the full frequency information, you'd also want to keep track of the y-coordinate, but more on that below.
+
+So this little mathematical machine does exactly what we wanted: it pulls out the constituent frequencies from a provided waveform, essentially unmixing a mixed bucket of paint. The applet below allows you to change the input wave form by separating the frequencies by commas.
+
+<Interactive filename="advanced_interactive/index" aspectRatio={880 / 500} />
+
+What happens when an input frequency of $0$Hz is wound up?
```

Comment:
Is the dotted line supposed to represent a neutral pressure? Because if that's the case, should 0Hz be constant at that line?

Reply:
Yes the dotted line is supposed to represent neutral pressure, great catch! I decided to normalize the resulting wave because if you had a wave composed of 1+2+3+4 then it would be drawn partially off the screen because the amplitude of that wave is simply larger than a wave composed of a single frequency. But 0 Hz should not affect the normalized wave!

Reply:
I've thought about it some more and I actually am thinking that it is actually fine the way it is. A signal with 0Hz is just a constant line, which could mean that it is always at the peak instead of always at neutral pressure.